.vscode/settings.json

```diff
@@ -24,7 +24,11 @@
     "statusBar.foreground": "#ffffff",
     "statusBar.background": "#1292EE",
     "statusBar.noFolderBackground": "#1292EE",
-    "statusBar.debuggingBackground": "#1292EE"
+    "statusBar.debuggingBackground": "#1292EE",
+    "titleBar.activeBackground": "#00b3e6",
```

Comment:
@tinkertrain  Hmm? Not sure where this is coming from 😅

Can we remove it? It looks a little disruptive for me

Reply:
I wonder if we should just removed settings.json from git. There is local customizations that we might want to do (per developper) and it should not be on the repo. If there is a way to separate local vs repo settings that would be even better

Reply:
I was about to make the same suggestion as @plbabin, I don't think editor settings should live in the repo, as they're or can be personal, people use different editors anyway. I didn't really intended to commit this change, I'm using this vscode extension that made the change https://marketplace.visualstudio.com/items?itemName=johnpapa.vscode-peacock

Reply:
@tinkertrain + @plbabin Sure :) We can remove it. Especially if folks are experience some pain with it.

I'll create a PR for that right now.

Reply:
https://github.com/helpscout/hsds-react/pull/565

Reply:
👏 Thanks!